server/db/svarog-11.psql

```diff
@@ -0,0 +1,34 @@
+-- This schema version changes the config columns from varchar to jsonb.
+
+ALTER TABLE observations ALTER COLUMN config TYPE jsonb USING json_build_object('text',config);
+ALTER TABLE stations ALTER COLUMN config TYPE jsonb USING json_build_object('text',config);
+
+-- Now we need to migrate existing data. This is really specific to station 1.
+-- Let's add protocol transmission. So far we were able to receive APT and LRPT.
+update observations set config = config || '{"proto": "APT" }' where sat_id in (25338, 28654, 33591) AND station_id = 1;
+update observations set config = config || '{"proto": "LRPT" }' where sat_id in (40069, 44387) AND station_id = 1;
+
+-- All transmissions received so far were done using crossed dipole
+update observations set config = config || '{"antenna": "WiMo TA-1", "antenna-type": "crossed dipole" }' WHERE station_id = 1;
+
+-- Add frequencies. We haven't done any Doppler magic, so everything was received on base frequencies.
+update observations set config = config || '{"freq": 137.62 }' WHERE sat_id = 25338 and station_id = 1; -- NOAA 15
```

Comment:
It should be "frequency". If we will promote the abbreviation then one user will use "freq" and another "f".